files/usmqe_alerts_logger.py

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/env python
+"""Tendrl Notifier REST API logger."""
+
+import argparse
+import ConfigParser
+import logging
+import os
+import json
+import time
+
+import requests
+
+
+URL = "http://0.0.0.0/api/1.0/"
```

Comment:
Could we have this url loaded from the config file as well? One problem here is that if ssl is enabled, this url may or may not work. The other thing is that I would like to have the script installed on the client machine.